ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -2821,6 +2857,95 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: '@yearly'
+  decorate: true
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: aws
+    job-release: "4.3"
+    job-test: e2e
+    job-type: upgrade
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-e2e-aws-upgrade-fips-4.3
```

Comment:
this job is to ensure that an upgrade from fips-enabled cluster will result in fips-enabled cluster

Reply:
Copying this will do nothing.  You need to copy the variants that take N-1 and upgrade to N

Reply:
What is this job testing? https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml#L2753-#L2823